training/distributed_training/pytorch/model_parallel/vit/submit_trainer_vit.ipynb

```diff
@@ -0,0 +1,566 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #28.                        "fp16_params": SM_HP_MP_PARAMETERS["fp16"],
</pre><p>This is deprecated, use <code>fp16</code> instead </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3572/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>